meson.build

```diff
@@ -135,7 +135,7 @@ preload_lib = shared_library('umockdev-preload',
    'src/ioctl_tree.c'],
   c_args: ['-fvisibility=default'],
   version: '0.0.0',
-  dependencies: [dl, pthread],
+  dependencies: [dl, glib, pthread],
```

Comment:
Please no -- this is a way too heavy and unpredictable dependency for the preload library.

Reply:
removed, hadn't used glib before my `libfprint` PR so was unaware of the cost